sapi/fuzzer/config.m4

```diff
@@ -24,12 +24,12 @@ if test "$PHP_FUZZER" != "no"; then
   SAPI_FUZZER_PATH=sapi/fuzzer
   PHP_SUBST(SAPI_FUZZER_PATH)
   if test -z "$LIB_FUZZING_ENGINE"; then
-    FUZZING_LIB="-lFuzzer"
+    FUZZING_LIB="-fsanitize=fuzzer"
```

Comment:
This would break oss-fuzz I assume. Not sure why thus was changed - this is a specific branch for LIB_FUZZING_ENGINE which is only used for oss-fuzz. Unless you add Fuzzer library somewhere else? I'll check the final build.

Reply:
This is the `-z` branch, used if `LIB_FUZZING_ENGINE` is not specified. The oss-fuzz case is in the else branch and not affected.

Reply:
oh sorry I misread it. My apologies.